droidlet/lowlevel/locobot/remote/slam_pkg/slam.py

```diff
@@ -90,16 +96,33 @@ def __init__(
         self.vis = vis
         self.save_vis = save_vis
         self.save_folder = save_folder
+        print(f"save_vis {save_vis}, save_folder {save_folder}")
         # to visualize robot heading
         triangle_scale = 0.5
         self.triangle_vertex = np.array([[0.0, 0.0], [-2.0, 1.0], [-2.0, -1.0]])
         self.triangle_vertex *= triangle_scale
         if self.save_vis:
-            self.save_folder = os.path.join(save_folder, str(int(time.time())))
+            self.save_folder = save_folder
+            self.img_folder = os.path.join(self.save_folder, "rgb")
+            self.depth_folder = os.path.join(self.save_folder, "depth")
+            self.seg_folder = os.path.join(self.save_folder, "seg")
             if not os.path.isdir(self.save_folder):
                 os.makedirs(self.save_folder)
+
+            if not os.path.isdir(self.img_folder):
+                os.makedirs(self.img_folder)
+
+            if not os.path.isdir(self.depth_folder):
+                os.makedirs(self.depth_folder)
+
+            if not os.path.isdir(self.seg_folder):
+                os.makedirs(self.seg_folder)
+        
+        self.last_pos = self.robot.base.get_state()
+
         self.start_vis = False
         self.vis_count = 0
+        self.skp =0
```

Comment:
nit:  spacing `= 0`

Reply:
is this better named as `self.skip_frames` or `self.save_vis_skip_frames`?

Reply:
renamed to `self.save_vis_skip_frames`